firebase/test/core/data/schemas/memo.spec.ts

```diff
@@ -5,88 +5,61 @@ import SerializationError from "#faults/errors/serialization-error";
 
```

Comment:
Updated to follow the same format as the one proposed in the `local-public-collection` file (reduce visual complexity)